frontend/.eslintrc

```diff
@@ -10,9 +10,8 @@
     "plugin:react/recommended",
     // Uses the recommended rules from the @typescript-eslint/eslint-plugin
     "plugin:@typescript-eslint/recommended",
-    // Uses eslint-config-prettier to disable ESLint rules
-    // from @typescript-eslint/eslint-plugin that would conflict with prettier
-    "prettier/@typescript-eslint",
```

Comment:
Why is this going away? Did it get superseded by the plugins below?

Reply:
Correct, all the `prettier/` configs were consolidated, so covered by `plugin:prettier/recommended`
Reference [link](https://github.com/prettier/eslint-config-prettier/blob/main/CHANGELOG.md#version-800-2021-02-21)